zulipterminal/ui_tools/views.py

```diff
@@ -331,18 +335,28 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
 
         elif is_command_key('REPLY_MESSAGE', key):
             self.body.keypress(size, 'enter')
+            if self.footer.focus is None:
+                return key
             self.set_focus('footer')
             self.footer.focus_position = 1
             return key
```

Comment:
Here the key is returned at the bottom and in the conditional; how about placing the other code into an opposite conditional and having the common return statement at the bottom?

Reply:
Yes done.